packages/@lightningjs/ui-components/src/components/Tile/Tile.d.ts

```diff
@@ -36,21 +38,91 @@ export type TileStyle = SurfaceStyle & {
 // TODO: Replace this with an official version from Lightning when one exists
 // type PatchTemplate<T extends lng.Element.Constructor> =
 //   lng.Element.PatchTemplate<lng.Element.ExtractTemplateSpec<InstanceType<T>>>;
-
-export default class Tile extends Surface {
-  // TODO: Replace value with PatchTemplate<typeof Artwork> when the Component template specs are created
-  artwork?: Record<string, unknown>;
+declare namespace Tile {
+  export interface TemplateSpec extends Surface.TemplateSpec {
+    Content: typeof lng.Component<lng.Component.TemplateSpecLoose>;
+    /**
+     * Object containing all properties supported in the [Artwork component](?path=/docs/components-artwork--artwork)
+     */
+    artwork: lightning.Element.PatchTemplate<Artwork.TemplateSpec>;
```

Comment:
I believe this how we are supposed to do these, given that `updateArtwork` is patching to Artwork